server/libs/infra/src/repositories/media.repository.ts

```diff
@@ -117,6 +117,7 @@ export class MediaRepository implements IMediaRepository {
             .output(output)
             .on('error', reject)
             .on('end', () => fs.unlink(`${output}-0.log`))
+            .on('end', () => fs.rm(`${output}-0.log.mbtree`, { force: true }))
```

Comment:
In general, whenever we write a file to a disk I think non-root users have to make changes for their app to continue to work, including kubernetes users. We should consider having a variable for temp locations, possible defaulted to `/tmp` where these files can be written to instead.